docs/install/install-node.rst

```diff
@@ -4,166 +4,99 @@
 Installing the Flocker Node Services
 ====================================
 
-It is possible to deploy Flocker in the cloud, on a number of different providers.
+.. _installing-flocker-node-prereq:
 
-- :ref:`Using Amazon Web Services <aws-install>`
-- :ref:`Using Rackspace <rackspace-install>`
+Prerequisites
+=============
 
-It is also possible to install Flocker on any CentOS 7 or Ubuntu 14.04 machine.
+Before you begin to install the Flocker node services, you will need the following:
 
-- :ref:`Installing on CentOS 7 <centos-7-install>`
-- :ref:`Installing on Ubuntu 14.04 <ubuntu-14.04-install>`
+* You will require a minimum of 2 nodes in order to install and use Flocker:
+  
+  * We support installing the Flocker node services on either :ref:`CentOS 7<centos-7-install>` or :ref:`Ubuntu 14.04<ubuntu-14.04-install>`.
+  * We recommend a minimum of 16GB storage on each node.
 
-.. _aws-install:
+* You will need permission for SSH access from your laptop.
+* Depending on your usage of Flocker, you will require access to a range of ports.
+  For example, specifying which ports to make available are included in the  :ref:`aws-install` documentation.
+* Flocker's container management features depend on Docker.
+  You will need to make sure `Docker (at least 1.8) is installed`_ and running.
 
-Using Amazon Web Services
-=========================
+Helpful Guides
```

Comment:
As this only contains guides for setting up nodes, the title should reflect that.